src/index.js

```diff
@@ -184,7 +196,8 @@ class App extends React.Component {
 
     renderInfo() {
         return (
-            <div className="info-page">
+            <div className="mainContainer">
```

Comment:
I am surprised no other changes is required because of this?

Reply:
That was because the .cs file had layout called "info-page" I simply reused "mainContainer" which was already made.